packages/oc-pages/vue_shared/client_utils/dashboard.js

```diff
@@ -0,0 +1,7 @@
+export function projectPathToHomeRoute(projectPath) {
+    const split = projectPath.split('/')
+    if(split.length == 2 && split[1] == 'dashboard') {
```

Comment:
is this making an assumption that breaks with multiple dashboard projects?

Reply:
No, it just handles the exception that /home/a10 would represent /a10/dashboard as opposed to /home/<project-path>